specification/loadtestservice/data-plane/loadtesting/routes.tsp

```diff
@@ -242,6 +256,53 @@ interface LoadTestRun {
   @operationId("LoadTestRun_StopTestRun")
   stop is StandardResourceOperations.ResourceAction<TestRun, {}, TestRun, {}>;
 
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @summary("Get the latest insights for the test run.")
+  @doc("Get the latest insights for the test run.")
+  @added(APIVersions.v2025_03_01_preview)
+  @route("/test-runs/{testRunId}/insights/latest")
+  @get
+  getLatestInsights is Azure.Core.Foundations.Operation<
+    {
+      @doc("""
+        Unique name for the load test run, must contain only lower-case alphabetic,
+        numeric, underscore or hyphen characters.
+        """)
+      @path
+      @maxLength(50)
+      @minLength(2)
+      @pattern("^[a-z0-9_-]*$")
+      testRunId: string;
+    },
```

Comment:
Can we extract this? Path param of `testRunId` is repeated in different APIs.

Reply:
Done, i have updated the swagger files as well for the previous version, hope thats fine?